github/pulls.go

```diff
@@ -134,6 +135,29 @@ func (s *PullRequestsService) Get(owner string, repo string, number int) (*PullR
 	return pull, resp, err
 }
 
+// Get pull request raw format (diff or patch)
```

Comment:
To follow Go style, the comment should begin with the symbol name. It's `GetRaw`, so it should be `// GetRaw gets ...`. Also, it'd be nice to end the sentence with a period.

> Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.

Source: https://golang.org/doc/effective_go.html#commentary